ivy/functional/ivy/device.py

```diff
@@ -689,15 +689,16 @@ def gpu_is_available() -> bool:
 
     Returns
     -------
-    ret
-        Boolean, as to whether a gpu is available.
+    ret: bool
```

Comment:
Hi @Boghdady9, we don't add typehints here in the docstring. You should remove this.